packages/install-expo-modules/src/plugins/cli/withCliIntegration.ts

```diff
@@ -65,10 +65,28 @@ const withCliBabelConfig: ConfigPlugin = (config) => {
   return withDangerousMod(config, [
     'ios',
     async (config) => {
-      const babelConfigPath = path.join(config.modRequest.projectRoot, 'babel.config.js');
-      let contents = await fs.promises.readFile(babelConfigPath, 'utf8');
-      contents = updateBabelConfig(contents);
-      await fs.promises.writeFile(babelConfigPath, contents);
+      const babelConfigJsPath = path.join(config.modRequest.projectRoot, 'babel.config.js');
+
+      if (await fs.promises.exists(babelConfigJsPath)) {
```

Comment:
`fs.promises` doesn't have the `exists` function: https://nodejs.org/api/fs.html#promises-api.

Reply:
Fixed, thank you!